amnestywebsite/humanity-theme
# Remove Unused Hero Pattern

Ref: https://github.com/amnestywebsite/humanity-theme/issues/246

Removes an unused pattern.

**Steps to test**:
1. Edit a post and open the pattern inserter
2. Search Hero
3. The pattern should no longer appear

Example: http://bigbite.im/i/kSfNBW
